src/app/Instance/InstancesListPage/InstancesListPage.tsx

```diff
@@ -128,25 +173,37 @@ const InstancesListPage = () => {
           </Text>
         </TextContent>
       </PageSection>
-      <PageSection>
-        <Button onClick={() => setShowCreateInstance(true)}>
-          {t("instance.createSEInstance")}
-        </Button>
-        <CreateInstance
-          isLoading={false}
-          isModalOpen={showCreateInstance}
-          onClose={() => setShowCreateInstance(false)}
-          onCreate={() => setShowCreateInstance(false)}
-        />
-      </PageSection>
-      <PageSection>
+      <PageSection
+        style={{
+          backgroundColor: "var(--pf-global--BackgroundColor--100)",
+          backgroundClip: "content-box",
+        }}
+      >
         <Table
+          caption={
+            <section>
+              <Button onClick={() => setShowCreateInstance(true)}>
+                {t("instance.createSEInstance")}
+              </Button>
+              <CreateInstance
+                isLoading={false}
+                isModalOpen={showCreateInstance}
+                onClose={() => setShowCreateInstance(false)}
+                onCreate={() => setShowCreateInstance(false)}
+              />
+              {getPagination(true)}
+            </section>
```

Comment:
I see you are using the caption for the controls. There's nothing technically wrong with it but I think a PF Toolbar is a better fit. [Here's an example](https://www.patternfly.org/v4/components/table/react-demos#bulk-select).
It will also take care of the correct spacing and responsiveness.

Reply:
I don't see too much value in using `<Toolbar>` right now:
![image](https://user-images.githubusercontent.com/22591802/161607417-6a88172f-9f4a-4a7a-a8d2-b3ec68c0a69a.png)
As you can see, I need to add padding on the left and the right. `<caption>` seems quite a more natural solution for this case.